src/Arcus.Messaging.Tests.Unit/ServiceBus/AzureServiceBusMessagePumpSettingsTests.cs

```diff
@@ -58,33 +52,7 @@ public void CreateSettings_WithTokenCredential_Succeeds()
                 options: options,
                 serviceProvider: serviceProvider);
         }
-        
-        [Fact]
-        public async Task GetEntityPath_WithNamespaceConnectionStringInsteadOfEntityScopedUsingConnectionString_Fails()
```

Comment:
Test was checking if we fail if the connection string is not including an entity name, but that is now being handled further up in the settings, not available to be checked here.